vwifi.c

```diff
@@ -56,6 +67,7 @@ struct ap_info_entry_t {
     u8 bssid[ETH_ALEN];
     char ssid[SSID_MAX_LENGTH];
 };
+static struct owl_context *g_ctx[2] = {NULL};
```

Comment:
It is a bit confusing to declare `g_ctx` as an array. Why would you do this?

Reply:
My demo scenario is using two stations connected to AP and ping each other.
![image](https://user-images.githubusercontent.com/5992038/162579001-f37887b7-4d14-43d2-b2c7-e7e8f5b9d515.png)
Create two stations which mean need two contexts.

Reply:
Maybe we also need to implement `.add_virtual_intf` and `.del_virtual_intf` which from `struct cfg80211_ops` . It could use the same connect link.

Reply:
I dislike the way to introduce the array of `g_ctx` because
1. It is not elegant: there is no evident relationship between `owl` network interfaces and their context (abbreviation: `ctx`).
2. The original naming scheme, `g_ctx`, refers to the global context that `vwifi` should maintain. However, we should initialize one network interface one time rather than pre-allocated and fixed contexts.
3. It does not make sense to have the restriction about the number of available `owl` network interfaces.

Reply:
ok, I will add `struct cfg80211_ops{.add_virtual_intf, .del_virtual_intf }` implementation to support multiple network interfaces.

Reply:
I misunderstand the `add_virtual_intf` meaning. It becomes more complicated for vwifi.
The `add_virtual_intf` will use the following command to bring up the interface. It is not my original thought.
`sudo iw phy owl interface add owl0 type station`
I will use more easy ways such as module parameters to decide how many network interfaces.
and add a link list into 'owl_context' to manage the multiple `net_device`.

Reply:
> I will use more easy ways such as module parameters to decide how many network interfaces. and add a link list into 'owl_context' to manage the multiple `net_device`.

I agree with the way to utilization parameters.